src/components/link-preview/link-preview.stories.tsx

```diff
@@ -1,9 +1,19 @@
 import React from "react";
-import { ComponentStory } from "@storybook/react";
+import { Meta, StoryObj } from "@storybook/react";
+
 import { action } from "@storybook/addon-actions";
+
 import LinkPreview from ".";
 
-export const DefaultStory: ComponentStory<typeof LinkPreview> = () => {
+const meta: Meta<typeof LinkPreview> = {
+  title: "Link Preview",
+  component: LinkPreview,
```

Comment:
**comment**: I assume an oversight, but we need an`argTypes` here with some styled system props that were generated with `margin: true`, since in the previous version we had `<StyledSystemProps margin />`. The docs for this component are indeed missing the styled system props table.